docs/tasks/text-to-image.md

```diff
@@ -29,7 +29,7 @@ From now on, all the action happens in the cloud!
 First you need to {ref}`login to Jina ecosystem <login-to-jina-ecosystem>`:
 ```python
 import finetuner
-finetuner.login()
+finetuner.login()  # use finetuner.notebook_login() in Jupyter notebook or Google Colab
```

Comment:
Can you update this tutorial with the correct da names? These are `fashion-train-data-clip` and `fashion-eval-data-clip`